### DISCUSSION TITLE
help logging in

### DISCUSSION THREAD

**Discussion by tvanicraath** [Dan6erbond/jolt#12] [Q&A]
Hi! Can you please help accessing the logs?

I am trying to set up jolt in a docker container and connect it to jellyfin. I have jellyfin instance set up and running, and is accessible from jolt container at http://jellyfin:8096. (I've been using jellyfin and jellyseerr since a month or so and they are working fine.) I added this address in the config.yaml file. But when I try to log in, there is no visual feedback and no message in docker logs. Can you help me finding out what is wrong?

Thanks,

**Comment by Dan6erbond** [comment]
Hi! The Docker logs should be as detailed as possible. Did you check the browser logs in case something failed with the connection? I currently only tested with an HTTPS instance of Jellyfin, but can't imagine the connection to be the issue. 

You can also try logging in with the playground by going to the `/playground` endpoint, and pasting the following mutation in the editor:

```gql
mutation SignInWithJellyfin {
  signInWithJellyfin(input: { username: "", password: "" }) {
   accessToken
    refreshToken
  }
}
```

Since Jolt is still in very early development stages, a lot of this stuff is untested. You might also want to try updating your image to the `main` tag which is currently the latest based on the `main` branch.

**Comment by tvanicraath** [comment]
Thanks for pointing out the HTTPS part. I can now sign in :-) Though soon after signing in, I got multiple "FATAL:  role "root" does not exist" errors and couldn't do anything. I will try to figure out what went wrong.

Thanks!
